programs/manifest/src/quantities.rs

```diff
@@ -325,23 +325,25 @@ impl QuoteAtomsPerBaseAtom {
         }
     }
 
-    pub fn multiply_spread(self, spread_e_5: u32) -> Self {
+    pub fn multiply_spread(self, spread_e_decimals: u32, decimals: u32) -> Self {
```

Comment:
spread mantissa / spread exponent? it's not clear what spread_e_decimals should mean